torchchat/utils/gguf_loader.py

```diff
@@ -24,6 +24,9 @@
     pack_scales_and_zeros,
 )
 
+from torchao.dtypes.utils import is_device
+from torchao.utils import TORCH_VERSION_AT_LEAST_2_6
```

Comment:
torchchat locks onto a specific torch version, so we don't need to check

Assume > 2.6

Reply:
The CI failures seem that torchao version is not that new, because TORCH_VERSION_AT_LEAST_2_6 is a new one. And I saw you pin pytorch nightly to 20241013, which is also not new, and this nightly does not have https://github.com/pytorch/pytorch/pull/139611 inside. This is my question, because the nightly used in the CI is 20241126.

Reply:
yup, working on the bump here: https://github.com/pytorch/torchchat/pull/1367

We'll test your fixes on there

Reply:
Thanks!